Src/DependencyCollector/Shared/Implementation/FrameworkHttpEventListener.cs

```diff
@@ -78,13 +78,23 @@ protected override void OnEventWritten(EventWrittenEventArgs eventData)
                 switch (eventData.EventId)
                 {
                     case BeginGetResponseEventId:
-                        this.OnBeginGetResponse(eventData);
+                        if (!DependencyTableStore.Instance.IsDesktopHttpDiagnosticSourceActivated)
+                        {
+                            // request is handled by Desktop DiagnosticSource Listener
+                            this.OnBeginGetResponse(eventData);
```

Comment:
Lets keep the logging here to indicate begin is skipped due to DiagSource.
DependencyCollectorEventSource.Log.SkipTrackingTelemetryItemWithEventSource(id);